src/mms_client/types/offer.py

```diff
@@ -94,7 +66,7 @@ class OfferStack(Payload):
     tertiary_2_qty_kw: Optional[int] = power_positive("Tertiary2OfferQuantityInKw", True)
 
     # The unit price of the power, in JPY/kW/segment
-    unit_price: Decimal = price("OfferUnitPrice")
+    unit_price: Decimal = price("OfferUnitPrice", 10000.00)
```

Comment:
Isn't the second argument of price the bool "optional"? What is going on here passing it 10,000?

Reply:
No, the second argument is a floating-point value describing the upper limit of the field.